src/components/Button/Button.module.scss

```diff
@@ -1,11 +1,12 @@
 .button {
-  @apply rounded px-4 py-2 text-sm font-bold;
+  @apply w-[296px] h-[40px] rounded-lg px-4 py-2 text-sm font-bold;
```

Comment:
please avoid giving the exact pixels style. it will create problem in responsive design.